__tests__/unit/NearbyInterest.test.js

```diff
@@ -4,24 +4,34 @@ import renderer from "react-test-renderer";
 import { shallow } from "enzyme";
 
 describe("Nearby interest component", () => {
+
+  const navigation = {
+    getParam: (param, defaultValue) => {
+      return "Tim Hortons"
+    }
+  };
+
+  test("renders correctly", () => {
+    const tree = renderer.create(<NearbyInterest navigation={navigation}/>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+});
+
+describe("Nearby interest component", () => {
+
   let wrapper;
   const setState = jest.fn();
   const useStateSpy = jest.spyOn(React, "useState");
   useStateSpy.mockImplementation((init) => [init, setState]);
 
-
   beforeEach(() => {
     wrapper = shallow(<NearbyInterest />);
   });
```

Comment:
<img width="763" alt="Screen Shot 2020-04-18 at 1 18 02 PM" src="https://user-images.githubusercontent.com/35850933/79644442-0a2c7b00-8177-11ea-8b76-8f2273ccff9c.png">

Reply:
I didn't touch that file. I think it would go on the refactor.

Reply:
> I found a small bug. When searching Men's/Women's Washroom or Water Fountain (Indoor POI), the app responds with:
> 
> `null is not an object (evaluating 'searchItem.coordinates.latitude')`

It could be added to the DoubleSearch search bars manually instead, just like we did with current location and point of interest, and removed as soon as we pressed on the back button.